pkg/cmd/flags.go

```diff
@@ -85,25 +88,30 @@ func (sv *OptionalServiceVersion) Parse(sid string, c api.Interface) (*fastly.Ve
 // identify if a given service version needs to be cloned.
 type OptionalAutoClone struct {
 	OptionalBool
+	Out    io.Writer
+	Client api.Interface
 }
 
 // Parse returns a service version.
 //
 // The returned version is either the same as the input argument `v` or it's a
 // cloned version if the input argument was either active or locked.
-func (ac *OptionalAutoClone) Parse(v *fastly.Version, sid string, c api.Interface) (*fastly.Version, error) {
+func (ac *OptionalAutoClone) Parse(v *fastly.Version, sid string) (*fastly.Version, error) {
 	// if user didn't provide --autoclone flag
 	if !ac.Value && (v.Active || v.Locked) {
 		return nil, fmt.Errorf("service version %d is not editable", v.Number)
 	}
 	if ac.Value && (v.Active || v.Locked) {
-		version, err := c.CloneVersion(&fastly.CloneVersionInput{
+		version, err := ac.Client.CloneVersion(&fastly.CloneVersionInput{
 			ServiceID:      sid,
 			ServiceVersion: v.Number,
 		})
 		if err != nil {
 			return nil, fmt.Errorf("error cloning service version: %w", err)
 		}
+		msg := fmt.Sprintf("Service version %d is not editable, so it was automatically cloned because --autoclone is enabled. Now operating on version %d.", v.Number, version.Number)
+		text.Output(ac.Out, msg)
+		text.Break(ac.Out)
```

Comment:
Shouldn't this be limited to when verbose output was requested?

Reply:
Agreed. Will modify